art/attacks/evasion/imperceptible_asr/imperceptible_asr_pytorch.py

```diff
@@ -147,13 +147,7 @@ def __init__(
         import torch  # lgtm [py/repeated-import]
         from torch.autograd import Variable
 
-        if (
-            hasattr(estimator, "preprocessing")
-            and (estimator.preprocessing is not None and estimator.preprocessing != (0, 1))
-        ) or (
-            hasattr(estimator, "preprocessing_defences")
-            and (estimator.preprocessing_defences is not None and estimator.preprocessing_defences != [])
-        ):
+        if estimator.preprocessing:
```

Comment:
Are we working to enable the preprocessing of signals?

Reply:
The former `preprocessing_defences` is being integrated into `preprocessing` with the goal to have only one preprocessing abstraction and not distinguish between preprocessing and preprocessing defences anymore. Based on offline discussion with @minhitbk we don't need this check anymore and I'll remove it with a new commit.